tests/integration_tests/test_filters.py

```diff
@@ -0,0 +1,393 @@
+import pytest
```

Comment:
It would be nice if we could start implementing unit tests instead of integration tests for pieces of code like the "filters parser".

I don't think this is blocking for the PR but we should make tickets for adding unit tests and starting working on them.